packages/react/src/AnchorNav/AnchorNav.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import {ComponentMeta} from '@storybook/react'
+import {AnchorNav} from '.'
+import {Heading, Text, Stack} from '../'
+import {RedlineBackground} from '../component-helpers'
+
+type MockData = {
+  [key: string]: string
+}
+
+export default {
+  title: 'Components/AnchorNav',
+  component: AnchorNav,
+  parameters: {
+    layout: 'fullscreen'
+  },
+  args: {
+    enableDefaultBgColor: true,
+    data: {
+      ['GitHub vs Jenkins']: 'githubvsjenkins',
+      ['GitHub vs GitLab']: 'githubvsgitlab',
+      ['GitHub vs CircleCI']: 'githubvscircleci',
+      ['GitHub vs BitBucket']: 'githubvsvsbitbucket',
+      ['GitHub vs TravisCI']: 'githubvsvstravis'
+    },
+    offset: 100
+  },
+  argTypes: {
+    data: {
+      name: 'Data',
+      description: 'Test data',
+      control: {
+        type: 'object'
+      },
+      table: {
+        category: 'Story customization'
+      }
+    },
+    offset: {
+      name: 'Offset',
+      description: 'Offset from the top of the page',
+      control: {
+        type: 'number'
+      },
+      table: {
+        category: 'Story customization'
+      }
+    }
+  }
+} as ComponentMeta<typeof AnchorNav>
+
+export const Default = ({data, ...args}: {data: MockData}) => {
+  return (
```

Comment:
Is it typical that the default story doesn't show the sections that are scrollable to? I would imagine if this is where are users are likely to end up while in Storybook first then we might want to display that?

Reply:
I guess having it is Playground is pretty close though so up to you whatever you think is best.

Reply:
Yes, that was intentional to keep the story source restricted to only the component example. All other examples show in-situ usage, particularly Playground. The sections themselves aren't part of the AnchorNav, so shouldn't go in default.

Reply:
Ok maybe we should create info around this because some composable pieces like a card component might have no content in an example like this. Maybe that is ok but we should just make sure we standardize.

Reply:
I think the [ADR](https://github.com/github/primer/blob/main/adrs/2022-10-07-storybook-lookbook-story-format.md#default) captures this, or do you mean something else?

Could you give an example for the card you mentioned? Cards would typically work fine in isolation, won't they?